### ISSUE TITLE
Pendulum- V0 not converge

### ISSUE THREAD

**Issue by 1900360** [nslyubaykin/parallel_ppo#1]
When I run the Pendulum- V0 environment, the reward curve is as follows, which does not converge. Do you know why?
![image](https://user-images.githubusercontent.com/58902376/193452911-534a6130-cb5e-47ee-bf2f-8a5140844a62.png)

Code show as below：
[parallel_ppo.txt](https://github.com/nslyubaykin/parallel_ppo/files/9692436/parallel_ppo.txt)

**Comment by nslyubaykin** [comment]
Hi @1900360!

In fact, this is not technically an issue, as typically every environment requires separate hyper-parameters tuning. 

I was able to achieve solid performance with this set of parameters:

```python
# Define PPO actor:
actor = PPO(
    policy_net=NormalMLP(obs_dim, acs_dim, 
                         nunits=32, nlayers=2,
                         out_activation=torch.nn.Tanh(),
                         acs_scale=2, # !Note: to match action space borders!
                         init_log_std=-1.5),
    device=torch.device('cpu'),
    learning_rate=1e-4, 
    n_epochs_per_update=50, 
    batch_size=5000,
    dataloader_n_workers=0,
    grad_norm_clipping=None,
    target_kl=0.2, 
    eps=0.2,
    ent_coef=0.0,
    gamma=0.9, 
    standardize_advantages=True,
    weight_decay=0.0
)

# Define GAE-$ \lambda $ critic:
critic = GAE(
    critic_net=VMLP(obs_dim, nlayers=2, nunits=32),
    device=torch.device('cpu'),
    learning_rate=1e-4, 
    batch_size=5000,
    gamma=0.9, 
    gae_lambda=0.95,
    n_target_updates=1, 
    n_steps_per_update=50
)
```
Resulting Training Run:

![ppo_run_pendulum](https://user-images.githubusercontent.com/67604207/193569458-471ddbe0-36fd-414b-a8f3-1786d9a23c8f.png)

Resulting Policy:

https://user-images.githubusercontent.com/67604207/193569502-57b1334d-5313-4ac1-b8c7-7837da792554.mp4

**Comment by 1900360** [comment]
Hi @nslyubaykin！
I've achieved a similar upward trend in rewards, thanks for your help!
![image](https://user-images.githubusercontent.com/58902376/193580913-6ce7c637-f29d-4010-b2f8-505f824df0e0.png)

**Comment by nslyubaykin** [comment]
Nice! So, I guess, I will close the issue then!

P.S. Also, at this early stage of a project every positive feedback would be higly appreciated, ⭐or recomenadtion to fellow RL enthuseasts! 🙂
